test/models/maintenance_tasks/runner_test.rb

```diff
@@ -32,6 +32,18 @@ class RunnerTest < ActiveSupport::TestCase
       end
     end
 
+    test "#run yields the Run when there is no active Run (undocumented)" do
```

Comment:
Can we add a comment above each of these two tests instead of putting `undocumented` in the test name to make things clearer? Maybe:

`# Yielding the run is undocumented and not supported in the Runner's API`